skeleton.html

```diff
@@ -2,25 +2,24 @@
 <html lang="en">
 
 <head>
-    <title>Lambda Theta Delta | Fraternity | Irvine</title>
```

Comment:
Updates to skeleton so future users will have to fill these out

Reply:
I was wondering what skeleton was for...

Reply:
Skeleton so if we want to add a new page, we have a skeleton html layout ready to use

No need to copy paste a previous page delete a ton of data